flac.md

```diff
@@ -82,13 +84,15 @@ FLAC uses four methods for modeling the input signal:
 
 # Residual Coding
 
-FLAC currently defines two similar methods for the coding of the error signal from the prediction stage. The error signal is coded using Rice codes in one of two ways:
+FLAC uses Exponential-Golomb (a variant of Rice) coding as it's residual encoder. [You can learn more about Exp-Golomb coding on Wikipedia](https://en.wikipedia.org/wiki/Exponential-Golomb_coding)
```

Comment:
I would say that exp-golomb is Exponential-Golomb (and not capitalise the «abbreviation»).